wfe/web-front-end.go

```diff
@@ -41,8 +41,6 @@ const (
 	IssuerPath     = "/acme/issuer-cert"
 	BuildIDPath    = "/build"
 
-	// StatusRateLimited is not in net/http
-	StatusRateLimited = 429
 	// statusBadNonce is used as a sentinel value to force sendError to send
```

Comment:
Can we fix this statusBadNonce sentinel value now? Happy to do it in a separate change if so.